specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -566,49 +566,16 @@
         }
       }
     },
-	"/skillsets('{skillsetName}')": {
-      "get": {
-        "tags": ["Skillsets"],
-        "operationId": "Skillsets_Get",
-        "x-ms-examples": {
-          "SearchServiceGetSkillset": { "$ref": "./examples/SearchServiceGetSkillset.json" }
-        },
-        "description": "Retrieves a cognitive skillset in an Azure Search service.",
-        "externalDocs": {
-          "url": "https://docs.microsoft.com/rest/api/searchservice/get-skillset"
-        },
-        "parameters": [
-          {
-            "name": "skillsetName",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "description": "The name of the skillset to retrieve."
-          },
-          {
-            "$ref": "#/parameters/ClientRequestIdParameter"
-          },
-          {
-            "$ref": "#/parameters/ApiVersionParameter"
-          }
-        ],
-        "x-ms-request-id": "request-id",
-        "responses": {
-          "200": {
-            "description": "The skillset is successfully returned.",
-            "schema": {
-              "$ref": "#/definitions/Skillset"
-            }
-          }
-        }
-      },
+    "/skillsets('{skillsetName}')": {
       "put": {
-        "tags": ["Skillsets"],
+        "tags": [
+          "Skillsets"
+        ],
         "operationId": "Skillsets_CreateOrUpdate",
         "x-ms-examples": {
           "SearchServiceCreateOrUpdateSkillset": { "$ref": "./examples/SearchServiceCreateOrUpdateSkillset.json" }
         },
-        "description": "Creates a new cognitive skillset in an Azure Search service.",
+        "description": "Creates a new Azure search skillset or updates a skillset if it already exists.",
```

Comment:
@arv100kri There was a fair amount of back-and-forth about the branding of Cognitive Search and how it should be reflected here. Please make sure Luis is ok with this change.

Reply:
I wasn't aware of that... let me rephrase keeping the original intent intact